### DISCUSSION TITLE
Monaco-Editor example with module.scss

### DISCUSSION THREAD

**Discussion by benebsiny** [vercel/next.js#16226] [Help] [answered]
I ran the example https://github.com/vercel/next.js/tree/canary/examples/with-monaco-editor and it worked. But in the example, I can't use module.scss to render other components. 
It showed the message:
```
error - ./pages/index.module.scss 1:0
Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> .text {
|     color: red;
|     font-size: 22px;
```
What should I do?

**Comment by idirmosh** [comment]
make sure you have sass installed

```
npm install sass
```

**Reply by benebsiny** [reply]
I've installed it. But it showed the same message.

**Comment by benebsiny** [comment, marked as answer]
Ok I found a solution. Use [@monaco-editor/react](https://github.com/suren-atoyan/monaco-react) instead of [react-monaco-editor](https://github.com/react-monaco-editor/react-monaco-editor), I can use it directly without any other settings. It works fine very much with next.js
